draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -263,8 +264,7 @@ Unique IDentifiers).  A UUID is 128 bits long, and requires no central
 registration process.
 
 The information here is meant to be a concise guide for those wishing
-to implement services using UUIDs as URNs {{RFC8141}}.  Nothing in this document
-should be construed to override the DCE standards that defined UUIDs.
+to implement services using UUIDs, as URNs {{RFC8141}} or otherwise.
```

Comment:
This language is from the old RFC, and I'm not sure it makes sense to leave in here.  If we're going to say that there is another document that takes precedence over this one then we should link to it and make it part of the formal spec, otherwise I think we should just take this reference to DCE standards out.